src/sdk/3.5/windowsservercore-ltsc2022/Dockerfile

```diff
@@ -0,0 +1,79 @@
+# escape=`
+
+ARG REPO=mcr.microsoft.com/dotnet/framework/runtime
+FROM $REPO:3.5-windowsservercore-ltsc2022
```

Comment:
Seeing this just made me realize the .NET FX images should be updates to reference the 3 part base images similar to https://github.com/dotnet/dotnet-docker/issues/2885

Reply:
Logged https://github.com/microsoft/dotnet-framework-docker/issues/811